include/vapor/SliceGridAlongPlane.h

```diff
@@ -0,0 +1,72 @@
+#pragma once
+
+#include <vapor/glutil.h>
+#ifdef Darwin
+    #include <OpenGL/gl.h>
+#else
+    #include <GL/gl.h>
```

Comment:
I don't know if the VDC library is technically linking against OpenGL.

Reply:
Fixed.